src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -89,7 +90,7 @@ internal partial class Binder
                 return CreateAnonymousFunctionConversion(syntax, source, conversion, isCast, destination, diagnostics);
             }
 
-            if (conversion.IsTuple || 
+            if (conversion.IsTupleLiteral || 
                 (conversion.Kind == ConversionKind.ImplicitNullable && source.Kind == BoundKind.TupleLiteral))
             {
                 return CreateTupleConversion(syntax, (BoundTupleLiteral)source, conversion, isCast, destination, diagnostics);
```

Comment:
Yes. I renamed a bunch of things formerly called "Tuple" to "TupleLiteral", but this one slipped through. There is a chance, there are some more.